common/src/main/java/org/opensearch/ml/common/input/nlp/TextDocsMLInput.java

```diff
@@ -25,7 +25,7 @@
  * ML input class which supports a list fo text docs.
  * This class can be used for TEXT_EMBEDDING model.
  */
-@org.opensearch.ml.common.annotation.MLInput(functionNames = {FunctionName.TEXT_EMBEDDING, FunctionName.SPARSE_ENCODING, FunctionName.SPARSE_TOKENIZE})
+@org.opensearch.ml.common.annotation.MLInput(functionNames = {FunctionName.TEXT_EMBEDDING, FunctionName.SPARSE_ENCODING, FunctionName.SPARSE_TOKENIZE, FunctionName.QUESTION_ANSWERING})
```

Comment:
I remember you plan to add new input/output type for QA model, will you add it in next commit?

Reply:
Do we still needs to add `FunctionName.QUESTION_ANSWERING` here?

Reply:
yeah good catch, will remove it